BrainFlowPlugin/Source/BrainFlowPlugin/BrainFlowPlugin.Build.cs

```diff
@@ -5,44 +5,30 @@
 
 public class BrainFlowPlugin : ModuleRules
 {
-	public BrainFlowPlugin(ReadOnlyTargetRules Target) : base(Target)
-	{
+    public BrainFlowPlugin(ReadOnlyTargetRules Target) : base(Target)
+    {
         PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
 
         PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore" });
 
         PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "Public"));
 
-        if(Target.Platform == UnrealTargetPlatform.Win32)
```

Comment:
is it not supported anymore?

Reply:
Unfortunately no. It was removed in UE 5.0.